katalog/curator/README.md

```diff
@@ -6,12 +6,12 @@ to managed retention of your infrastructure logs to a given value.
 
 ## Requirements
 
-- Kubernetes >= `1.10.0`
-- Kustomize >= `v1`
+- Kubernetes >= `1.14.0`
+- Kustomize >= `v3`
 
 ## Image repository and tag
 
-* Curator image: `python:3.6-alpine`
+* Curator image: `python:3.7-alpine`
```

Comment:
Is this correct? From the manifests I see that we're using `quay.io/sighup/curator:5.8.1_3.7-alpine_5.3.1` as image.